### DISCUSSION TITLE
Location of logs

### DISCUSSION THREAD

**Discussion by sripor62** [concourse/concourse#8964] [General]
Where can I get logs of Concourse?

**Comment by suhlig** [comment]
Depends on how you deploy Concourse. I use [ansible-concourse](https://github.com/troykinsella/ansible-concourse), which starts web and worker processes as systemd units. I can then use e.g. `journalctl -f -u concourse-worker.service`.

**Reply by sripor62** [reply]
I want to access build logs.Where can I find it?

**Reply by suhlig** [reply]
Ah, I misunderstood. Build logs are stored in the database and exposed to the user via web UI and [fly watch](https://concourse-ci.org/builds.html#fly-watch).

**Reply by sripor62** [reply]
Are they not forwarded to CloudWatch?

**Reply by suhlig** [reply]
I don't think so. In general Concourse distinguishes between 

1. build logs, to be consumed by the Concourse user, and 
2. system logs, to be consumed by the Concourse operator.

Build logs are exposed via web UI and cli. System logs are emitted by the concourse binary to stdout / stderr, and their final destination depends on the deployment method.

I am not familiar with AWS, but it looks as if you could wire your deployment to show **system logs** in CloudWatch, but not build logs.
